lib/config-all.json

```diff
@@ -204,6 +204,7 @@
     "img/crossorigin",
     "img/jpeg2000",
     "img/jpegxr",
+    "img/loading",
```

Comment:
just one more thing: rename the file from just "loading" to "lazyloading" Makes it clearer what the feature does and correpsonds to the property too.

Reply:
No problem. Renamed.